cmd/lakefs/cmd/run.go

```diff
@@ -131,7 +131,7 @@ var runCmd = &cobra.Command{
 
 			// Check if migration required only on postgres
 			migrationRequired := false
-			if dbParams.Type == kvpg.DriverName {
+			if dbParams.Type == kvpg.DriverName && len(dbParams.ConnectionString) > 0 {
```

Comment:
In case ConnectionString is missing, what is the desired behavior?

Reply:
Basically our only indication that we have a previous postgres DB is the connection string. If it doesn't exist we assume migration is either completed or not needed